src/backend/tests/fixtures/get-test-server.ts

```diff
@@ -38,8 +38,12 @@ export const getTestServer = async (
     S3_USER_UPLOADS_ACCESS_KEY_ID: minio.accessKey,
     S3_USER_UPLOADS_SECRET_ACCESS_KEY: minio.secretKey,
     S3_USER_UPLOADS_BUCKET_NAME: minio.bucket,
-    EMAIL: "example string",
-    EMAIL_PASS: "example string",
+    EMAIL_SERVICE: "example string",
+    EMAIL_FROM: "example string",
+    EMAIL_AUTH_USER: "example string",
+    EMAIL_AUTH_PASS: "example string",
+    EMAIL_HOST: "example string",
+    EMAIL_PORT: "example string",
```

Comment:
Nitpick: should these be example strings that look somewhat like what they're representing?
e.g. "example@test.com"?